share/functions/__fish_print_VBox_vms.fish

```diff
@@ -0,0 +1,28 @@
+function __fish_print_VBox_vms
+    set -l print_names true
+    set -l print_uuids true
+
+    if set -q argv[1]
+        if test $argv[1] = 'nouuids'
```

Comment:
Alternatively, to make the arguments position-independent (and allow both), you could use `contains -- nouuids $argv`.